src/Trixi.jl

```diff
@@ -37,9 +37,10 @@ using OffsetArrays: OffsetArray, OffsetVector
 using RecipesBase
 using Requires
 @reexport using StaticArrays: SVector
-using StaticArrays: MVector, MArray, SMatrix
+using StaticArrays: MVector, MMatrix, MArray, SMatrix
 using TimerOutputs: TimerOutputs, @notimeit, @timeit_debug, TimerOutput, print_timer, reset_timer!
-using UnPack: @unpack, @pack!
+@reexport using UnPack: @unpack
```

Comment:
What's the purpose for this reexport?

Reply:
I often want to use it when working with Trixi in more complex setups. It's basically the same as `export`ing `SVector`